src/awesome_map/AwesomeMap.js

```diff
@@ -795,7 +795,8 @@ define(function(require) {
                 translateY: Math.round(options.y),
                 scale: options.scale
             });
-            this._publishChangingEvents(this._currentTransformState, newState);
+            var event = { type: undefined };
```

Comment:
Is this just a "dummy" event to make sure _publishChangingEvents and friends are happy?  You might want to comment that if so, because the code reads a little strangely to me.